web-app/src/auth/auth.controller.ts

```diff
@@ -0,0 +1,27 @@
+import {
+  Controller,
+  Post,
+  Request,
+  UseGuards,
+  Get
+} from '@nestjs/common';
+import { AuthGuard } from '@nestjs/passport';
+import { AuthService } from './auth.service';
+
+@Controller('api/auth')
+export class AuthController {
+
+  constructor(private authService: AuthService) { }
+
+  @UseGuards(AuthGuard('local'))
+  @Post('/login')
+  async login(@Request() req) {
+    return this.authService.login(req.user);
+  }
```

Comment:
You can give info about the type expected for you param `user` like I did in the games controller:

```ts
async createNewGame(@Body() gameParams: { size?: number, fromFile?: boolean }, @Req() req: Request) {
  ...
}
```

Reply:
Same comment than Anthony here : https://github.com/marmelab/Hex/pull/47#discussion_r813921727